File: PgAce.App/Forms/PropertiesPane.vb
Imports System.Windows.Forms
Imports WeifenLuo.WinFormsUI.Docking

Namespace PgAce.App.Forms
    Public Class PropertiesPane
        Inherits DockContent

        Public Sub New()
            Text = "Properties"
            Dim propertyGrid = New PropertyGrid() With {
                .Dock = DockStyle.Fill
            }
            Controls.Add(propertyGrid)
        End Sub
    End Class
End Namespace

File: PgAce.App/Forms/PropertiesPane.cs
using System.Windows.Forms;
using WeifenLuo.WinFormsUI.Docking;

namespace PgAce.App.Forms
{
    public class PropertiesPane : DockContent
    {
        public PropertiesPane()
        {
            Text = "Properties";
            var propertyGrid = new PropertyGrid
            {
                Dock = DockStyle.Fill,
            };
            Controls.Add(propertyGrid);
        }
    }
}
